synapse/appservice/__init__.py

```diff
@@ -27,7 +28,7 @@
 logger = logging.getLogger(__name__)
 
 
-class ApplicationServiceState:
+class ApplicationServiceState(Enum):
```

Comment:
Silly question, but this change isn't visible to the module API right?

Reply:
not a silly question! As far as I can tell, no it's not (and I can't think of any interfaces where it'd make sense). :)